include/overlay021/pokedexencdata.h

```diff
@@ -0,0 +1,155 @@
+#ifndef POKEPLATINUM_POKEDEXENCDATA_H
+#define POKEPLATINUM_POKEDEXENCDATA_H
+
+#include "constants/species.h"
+
+#include "cell_actor.h"
+
+typedef struct {
+    int *locations;
+    int numLocations;
+} EncounterLocations;
```

Comment:
Make sure your structs have both the same canonical name and typedef'd name, e.g.,

```c
typedef struct EncounterLocations {
    int *locations;
    int numLocations;
} EncounterLocations;
```